.github/workflows/build.yml

```diff
@@ -19,11 +19,15 @@ jobs:
                 publish-phar: [false]
                 include:
                     -   php: "7.3"
+                        composer-version: "1"
                         coverage: "pcov"
                         publish-phar: true
                     -   php: "7.4"
                         composer-version: "2"
 
+        env:
+            COMPOSER_ROOT_VERSION: '0.13.99'
```

Comment:
is there any way this is loaded from the `.composer-root-version` instead? This would make it slightly easier to maintain since only one place would need to be updated

Reply:
The source command didn't work because this how an env var should be set https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable

Reply:
Maybe that's still possible then?

```
source .composer-root-version
echo "COMPOSER_ROOT_VERSION =${COMPOSER_ROOT_VERSION}" >> $GITHUB_ENV
```

Reply:
Yes, this works 🎉